src/Microsoft.SqlTools.ServiceLayer/Profiler/ProfilerSessionMonitor.cs

```diff
@@ -186,113 +185,99 @@ private bool RemoveSession(int sessionId, out ProfilerSession session)
             }
         }
 
-        public void PollSession(int sessionId)
-        {
-            lock (this.sessionsLock)
-            {
-                this.monitoredSessions[sessionId].pollImmediatly = true;
-            }
-            lock (this.pollingLock)
-            {
-                Monitor.Pulse(pollingLock);
-            }
-        }
-
         /// <summary>
-        /// The core queue processing method
+        /// The core queue processing method, cycles through monitored sessions and creates a stream for them if not already.
         /// </summary>
-        /// <param name="state"></param>
-        private void ProcessSessions()
+        private void ProcessStreams()
```

Comment:
Given that everything else is still referring to these as "sessions" I'd say just keep this as `ProcessSessions`